engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/SymbolResolutionTest.java

```diff
@@ -0,0 +1,137 @@
+package org.enso.interpreter.test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.OutputStream;
+import java.net.URI;
+import org.enso.polyglot.MethodNames;
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.Source;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class SymbolResolutionTest extends TestBase {
+  private static Context ctx;
+
+  @BeforeClass
+  public static void prepareCtx() {
+    ctx =
+        defaultContextBuilder()
+            .out(OutputStream.nullOutputStream())
+            .err(OutputStream.nullOutputStream())
+            .build();
+  }
+
+  @AfterClass
+  public static void disposeCtx() {
+    ctx.close();
+  }
+
+  @Test
+  public void resolvingLocalSymbol() throws Exception {
+    final URI uri = new URI("memory://resolvingLocalSymbol.enso");
+    final Source src =
+        Source.newBuilder(
+                "enso",
+                """
+                    my_symbol = 42
```

Comment:
Hmm, the test is passing on `develop` as well.

I've added it as part of development when I was modifying the logic in `BindingsMap`, in particular `resolveQualifiedName` and `resolveLocalName`. 

I was amending that to make sure that my tests can resolve the type names from the current module through a qualified name. I guess that is no longer really needed, but the changes were kept as they do make sense - before the `resolveQualifiedName` would fail for `current_module_name.Foo` IIRC. I think they used to be broken at least for a bit during development and I wanted to have these tests to ensure that my changes to the logic did not break anything.